examples/text/news_topic_classification.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
```

Comment:
What changed because of which this now converges?

Reply:
Oh the example was not running at all. Was erroring out. It wasn't on master probably because we started debugging locally before we ran it on your cluster.

Small changes here and there.

Reply:
oh great!